index.js

```diff
@@ -26,6 +26,9 @@ const chooseKey = key =>
 
 const decode = partialRight(jwt.decode, [{ complete: true }])
 
+const stripBearer = token => 
+  token ? token.replace(/^[B|b]earer /, '') : null
```

Comment:
The idiomatic way to do this in js is with the [case-insensitive `i` flag](https://devdocs.io/javascript/global_objects/regexp).  Also, Ramda has a [point-free `replace` function](https://devdocs.io/ramda/index#replace):

```js
const stripBearer =
  replace(/^Bearer /i, '')
```

The null-check is best left to the existing `enforce` function.  I'll make a separate comment explaining that.